packages/whenable/README.md

```diff
@@ -26,12 +26,14 @@ Here they are: {
 }
```

Comment:
Above

> a `watch` operator to subscribe to a promise [...] in a way that survives upgrades of both the creator and subscribing client vats.

implies that `watch` on a promise can also survive some upgrades. Which ones?

Reply:
Oh, but there is one way that the watch on a promise can survive: if the client vat is durable, a watch on a promise from a different server vat will survive an upgrade of the client.

Reply:
Yeah, it is worth being explicit about this space of choices.